### PR TITLE
FTR-84 registration flow is weird

### PR DESCRIPTION
**Pull request by ibeusan20**
FTR-84 Fixed weird registration flow. When user registers in the app and goes to the profile screen, his details aren't empty anymore. They are now visible and set to user specific default values.
Also quickly fixed logo on registration screen being black on dark background.

### CODE REVIEW

**Comment by ibeusan20** [comment]
@dpetek21 Feel free to merge this PR upon reviewing it.

**Comment by dpetek21** [comment]
Very good, sorry for delay, merry Christmas.

File: app/src/main/java/hr/foi/air/servicesync/backend/FirestoreUserDetails.kt [hunk 1]
```diff
@@ -17,6 +17,30 @@ class FirestoreUserDetails {
...
     var description: String = ""
 
+    fun registerUserDetails(
+        email: String,
+        password: String,
+        onResult: (Boolean) -> Unit
+    ) {
+        val userDetails = mapOf(
+            "email" to email,
+            "username" to email.substringBefore("@"),
```
Comment (mpuskadij) [inline app/src/main/java/hr/foi/air/servicesync/backend/FirestoreUserDetails.kt:27]: Nice method usage

**Review by mpuskadij — APPROVED** [review]
Not too noticable changes, very good usage of lambda functions.

**Pull request merged**